src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -221,7 +234,7 @@ def step(
         model_output: torch.FloatTensor,
         timestep: int,
         sample: torch.FloatTensor,
-        prediction_type: str = "epsilon",
```

Comment:
Actually I think we should deprecate this argument and just have it in the config instead - I don't it's something that differs for every scheduler call. @anton-l what do you think?

Reply:
That makes sense! That means the pipeline doesn’t have to change at all either

Reply:
I agree, the prediction type doesn't need to change during sampling